### ISSUE TITLE
[Driver] YCQL 4.X driver sets the consistency level to YB_CONSISTENT_PREFIX

### ISSUE THREAD

**Issue by srinivasa-vasu** [yugabyte/yugabyte-db#10113]
Jira Link: [DB-4642](https://yugabyte.atlassian.net/browse/DB-4642)
YCQL 4.x driver sets the consistency level to YB_CONSISTENT_PREFIX if it is not explicitly set at the statement level. This happens even if the local reads(follower reads => ConsistencyLevel.ONE) are not turned on. If the local read option is not on, it is expected to use the YB_STRONG consistency as the default and the connections are expected to be routed to the leader nodes. 
https://github.com/yugabyte/cassandra-java-driver/blob/1364973580a8d19b0b3ed0db022efc1353bea5a5/core/src/main/java/com/yugabyte/oss/driver/internal/core/loadbalancing/PartitionAwarePolicy.java#L187 
```    
private ConsistencyLevel getConsistencyLevel() {
      return statement.getConsistencyLevel() != null
          ? statement.getConsistencyLevel()
          : ConsistencyLevel.YB_CONSISTENT_PREFIX;
    }
```

This part always seems to set the consistency level to YB_CONSISTENT_PREFIX and the hosts' list always gets shuffled even for strong consistency calls.

**Comment by tedyu** [comment]
This seems related: https://github.com/yugabyte/cassandra-java-driver/pull/10

**Comment by ashetkar** [comment]
This is fixed for 4.6 in [this commit](https://github.com/yugabyte/cassandra-java-driver/commit/04381d489bc0965337f241f3b1b7c01f1cf1d84d). It's still an issue with 4.11-RC release but we are not planning to release newer versions of 4.11 since there is 4.14 version available from datastax which we plan to use going forward. Closing this ticket.
